pkg/features/kube_features.go

```diff
@@ -986,7 +987,7 @@ var defaultKubernetesFeatureGates = map[featuregate.Feature]featuregate.FeatureS
 
 	PreferNominatedNode: {Default: true, PreRelease: featuregate.GA, LockToDefault: true}, // remove in 1.26
 
-	ProbeTerminationGracePeriod: {Default: false, PreRelease: featuregate.Beta}, // Default to false in beta 1.22, set to true in 1.24
+	ProbeTerminationGracePeriod: {Default: true, PreRelease: featuregate.Beta}, // Default to true in beta 1.25
```

Comment:
@liggitt I think that this is not covered in https://github.com/kubernetes/enhancements/tree/master/keps/sig-architecture/3136-beta-apis-off-by-default

Reply:
It is a question, this was beta before the kep, it should be able to transition to enabled by default, no?

Reply:
yes, that KEP is only referring to beta endpoints / API versions

Reply:
(specifically called out in the non-goals - https://github.com/kubernetes/enhancements/tree/master/keps/sig-architecture/3136-beta-apis-off-by-default#non-goals)

> **Non-Goals**
>
> Change feature gate defaults. Feature gates control new features (not just new APIs) and they are on by default for beta features. This KEP is not changing the lifecycle flow for feature gates. It is currently alpha=off-by-default, beta=on-by-default, stable=locked-to-on.

Reply:
[3136-beta-apis-off-by-default](https://github.com/kubernetes/enhancements/tree/master/keps/sig-architecture/3136-beta-apis-off-by-default) only focus on the beta APIs, we can still change the default value of feature gates here, right? So there's no problem here.

Reply:
I was mixing endpoints and feature gates, forgive me